app/models/enumeration/EventType.scala

```diff
@@ -92,7 +92,7 @@ object EventType extends Enumerable.Implicits {
       case evType1832 if api1832Events.contains(evType1832) => Some(ApiType.Api1832)
       case evType1833 if api1833Events.contains(evType1833) => Some(ApiType.Api1833)
       case evType1831 if api1831Events.contains(evType1831) => Some(ApiType.Api1831)
-      case _ => None
+      case evType1834 if api1834Events.contains(evType1834) => Some(ApiType.Api1834)
```

Comment:
similarly we don't need this 
In `EventReportConnector.scala`, under `getEvent` method

`case None => throw new BadRequestException(s"No API is configured to handle getting this eventType: $eventType")`

Reply:
Addressed here but with the unchecked annotation - not sure if we'd like to proceed with this approach: https://github.com/hmrc/pension-scheme-event-reporting/commit/2c089af273be16623552a522c80200a00f9429a2

Reply:
here is simplified with last commit